test/integration/helpers.go

```diff
@@ -86,22 +87,44 @@ func (rr RunResult) Output() string {
 }
 
 // Run is a test helper to log a command being executed \_(ツ)_/¯
-func Run(t *testing.T, cmd *exec.Cmd) (*RunResult, error) {
+func Run(t *testing.T, cmd *exec.Cmd, powershell ...bool) (*RunResult, error) {
 	t.Helper()
-	rr := &RunResult{Args: cmd.Args}
+	runInPowershell := false
+	if len(powershell) > 0 {
+		runInPowershell = powershell[0]
+	}
+
+	var newCmd *exec.Cmd
+	if runInPowershell {
+		psBin, err := exec.LookPath("powershell.exe")
```

Comment:
This isn't necessary: The OS will do the path lookup for the first argument provided to exec.Command.